README.md

```diff
@@ -151,19 +160,34 @@ Then run the following commands:
 
 ```bash
 make install
-make docker-build
-make docker-push
+make docker-push IMG=<e.g. index.docker.io/nachtmaar/runtime-controller>
 make deploy
 ```
 
 ### Run the examples
 
+Create sample function
+
 ```bash
-kubectl apply -f config/samples/runtime_v1alpha1_function.yaml
+kubectl apply -f config/samples/runtime_v1alpha1_function.yaml -n REPLACE_BY_NAMESPACE
```

Comment:
Can we use `{}` or `<>` to indicate a variable? e.g. `{NAMESPACE}`

Reply:
Same for all `REPLACE_BY_NAMESPACE` occurrences

Reply:
I will use {NAMESPACE}